apps/ledger-live-desktop/src/renderer/screens/exchange/index.tsx

```diff
@@ -105,7 +107,9 @@ export type ExchangeComponentParams = {
 
 const Exchange = ({ match }: RouteComponentProps<ExchangeComponentParams>) => {
   const appId = match?.params?.appId;
+  const buySellUiFlag = useFeature("buySellUi");
```

Comment:
I'd rename this to be a little clearer for other developers ... explaining that this is a rollout on v3 of Buy Sell FE, maybe something like `buySellUIV3`

Reply:
if we make the manifest id extendable then the version could be anything rather than always 3, so it might be best to avoid using the version actually..

Reply:
if we make the manifest id extendable then the version could be anything rather than always 3, so it might be best to avoid using the version actually..